executor/index_lookup_join.go

```diff
@@ -360,9 +372,14 @@ func (ow *outerWorker) buildTask(ctx context.Context) (*lookUpJoinTask, error) {
 	task.memTracker.AttachTo(ow.parentMemTracker)
 
 	ow.increaseBatchSize()
+	task.outerResult.SetRequiredRows(ow.batchSize, ow.maxBatchSize)
```

Comment:
How about
```suggestion
	else {
		task.outerResult.SetRequiredRows(ow.batchSize, ow.maxBatchSize)
	}
```
to reduce a function call.

Reply:
updated.